android/sdl_android/src/androidTest/java/com/smartdevicelink/test/proxy/SystemCapabilityManagerTests.java

```diff
@@ -490,6 +490,129 @@ public void testAddOnSystemCapabilityListenerWithSubscriptionsNotSupportedAndCap
 		verify(internalInterface, times(1)).sendRPC(any(GetSystemCapability.class));
 	}
 
+	public void testAddOnSystemCapabilityListenerThenGetCapability() {
```

Comment:
```suggestion
	public void testAddOnSystemCapabilityListenerThenGetCapabilityWhenSubscriptionsAreNotSupported() {
```

You may also want to consider underscores to define conditions, like:

`testAddOnSystemCapabilityListener_thenGetCapability_whenSubscriptionsAreNotSupported()`

Reply:
The use of underscores in java is not very common. Except for naming const variables. I prefer the first suggested name as it complies more with Java naming conventions.